jupyterlab_dotscience/src/index.ts

```diff
@@ -110,10 +132,16 @@ const plugin: JupyterFrontEndPlugin<void> = {
       ]).then(results => {
         // update the commit list if it has changed
         const commitData = results[0]
-        
-        if(commitData.length!=COMMIT_DATA.length) {
-          COMMIT_DATA = commitData
-          populateCommits()
+
+        if (commitData) {
+          // remove the initial commit (not interesting/relevant to users) and reverse the
+          // list (so that more recent commits are shown at the top)
+          commitData.shift()
+          commitData.reverse()
+          if(commitData.length!=COMMIT_DATA.length) {
```

Comment:
if commitData were an object we were using elsewhere - we might want to say:

```
const commitData = [].concat(results[0])
```

i.e. take a copy of the array before mutating it so there are no side effects - I think this is the result of an api call however so is fine in this case :+1: